guides/0116-http_sink.md

```diff
@@ -0,0 +1,47 @@
+# http_sink
+
+* type: consumer
+* input: binary messages
+
+This is a consumer block that takes `data` from incoming `Message` and makes a POST request
+to the configured URL containing `data`.
+
+This block supports only incoming messages with type `:binary`, so serialization
+to binary format must be handled in a separate block before the message arrives here.
+
+The `subtype` of the message, if present, is added as `Content-Type` header.
+Additionally, static headers can be added to the POST requests with the initial configuration.
+For the time being, the delivery is best-effort (i.e. if a message is not delivered, it is discarded).
+
+# Properties
+
+* `url`: The URL of the HTTP request. (required, string)
+* `headers`: Http request headers. (optional, object with string values)
+* `ignore_ssl_errors`: If `true`, ignore SSL errors that happen while performing the request. (optional, boolean)
+
+## `url`
+
+The HTTP request URL such as `"http://www.example.com/test"`.
+
+## `headers`
+
+An optional object with additional headers such as `{authorization: "Bearer <token>"}`.
+
+# Input message
+
+The HTTP request built by the `http_sink` can be customized by the input message with the following proprietes:
+
+* `data`: the body of the request.
+* `type`: is always `binary`.
+* `subtype`: if present, it will be the value of the `content-type` HTTP header.
+
+# Examples
+
+The following example uses `http_sink` block to send HTTP POST requests to `http://www.example.com/test`
+whenever a valid message is received.
+
+```
+[...]
```

Comment:
at the moment we are not using [...] for code ellipsis, I like it. @rbino do you agree about using it?

Reply:
+1